CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributor Onboarding
+
+🚧 This document is still a WIP 🚧 
+
+## General
+
+ - The goal of this guide is to help you start contributing to Durable Functions
+
+## Pre-reqs
+
+ - OS
+    - Windows 10 (suggested)
+    - Mac OS X/Linux
+ - Language runtimes
+    - .NET Core 2.0
+ - Editor
+    - Visual Studio 2017 (recommended)
+    - VS Code 
+ - Misc tools (suggested)
+    - git - source control
+    - [Azure Storage Emulator](https://docs.microsoft.com/azure/storage/storage-use-emulator) or real Azure Storage Account
+
+## Testing code changes locally (Visual Studio) 
+
+Durable Functions are distributed as a NuGet package. So in order to test your changes, you need to integrate it in a function. For that you need the following steps:
+
+1. Create a local NuGet source on your dev machine - e.g. C:\LocalNuGet.
+2. Then add the local NuGet source as a NuGet package source in Visual Studio. 
+3. Modify the code.
+4. Build the project and this will create a new NuGet package with your changes.
+5. Add the newly built Durable Functions NuGet package to the local NuGet source. The command to add packages to this location is nuget **add package.nupkg -Source C:\LocalNuGet** . Add the symbols file as well to be able to debug your modifications while debugging the function code. 
```

Comment:
>Add the symbols file as well to be able to debug your modifications while debugging the function code.  [](start = 182, length = 103)

This should be automatic because of how the csproj is configured.